### DISCUSSION TITLE
How to make schema model accept multiple models as an attribute value ?

### DISCUSSION THREAD

**Discussion by meepeek** [prisma/prisma#7391] [Q&A]
My use case is financial security. I want to create schema where I can access the model Security which points to different type of securities. I simplified my use case as the following schema

```prisma
model Security {
  symbol  String
  market  String
  data      Stock | Fund | Index    <<-- this is what I want

  @@id([symbol, market])
}
model Stock {}
model Fund {}
model Index {}
```

The schema throws an error about not a valid field or attribute definition.

**Comment by ryands17** [comment]
@meepeek 👋 

This is currently not possible with Prisma, but we do have a feature request here: https://github.com/prisma/prisma/issues/2505

It would be great if you could add a 👍 to this so that we can look at the priority for this. The only workaround for now would be differentiating this in your application code or adding a type that saves the entity.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
